app/javascript/dashboard/components/SnackbarContainer.vue

```diff
@@ -1,15 +1,14 @@
 <template>
   <transition-group name="toast-fade" tag="div" class="ui-snackbar-container">
     <woot-snackbar
-      v-for="snackMessage in snackMessages"
-      :key="snackMessage"
+      v-for="(snackMessage, index) in snackMessages"
```

Comment:
@muhsin-k I think there might be some issue if you use index directly since we are constantly changing index with the below setTimeout. Can you check if this issue is valid?
Also you check if Date.now() is possible before pushing it on line number 34.